openquake/engine/engine.py

```diff
@@ -163,15 +163,11 @@ def raiseMasterKilled(signum, _stack):
         msg = 'The openquake master process was killed manually'
     else:
         msg = 'Received a signal %d' % signum
-
-    # FIXME this code has been temporary disabled due issues with large
-    # computations and further investigation is need; code is left as reference
-    # for pid in parallel.executor.pids:
-    #     try:
-    #         os.kill(pid, signal.SIGKILL)
-    #     except OSError: # pid not found
-    #         pass
-
+    for pid in parallel.executor.pids:  # when using futures
```

Comment:
I'm experiencing  issues with Python 2: `oq abort` does not do anything and this is preventing `CTRL-C` from work (and `SIGKILL` is needed. Everything is fine with Python 3: I would suggest to skip this block if Python 2.7 is used. Soon or later Python 2 support will be removed :)